### ISSUE TITLE
Broken simple line icons

### ISSUE THREAD

**Issue by chrismaree** [PangoSign/PangoSign#136]
some icons seem to be broken on search results, especially on hover

**Comment by CarlBeek** [comment]
It's definitely the commit you refer to that breaks things, but I'm not sure why. It is as though you are overriding the font in the .scss, but I can't find where in the diff between this and the previous commit.

**Comment by chrismaree** [comment]
@av80r I agree. will fix it

**Comment by CarlBeek** [comment]
It's even weirder that I initially anticipated. We have Simple Line Icons (SLI) rendering correctly elsewhere in the page. Take a look at the home page widgets for wallet balance etc.
I tried:

- adding and removing the meteor package
- just putting the SLI on their own without the popovers, just in the col-2
- adding imports for the font files the scss
- removing all the "fancy" classes we applied like text-muted etc.

None of the above worked.
